src/mon/Monitor.cc

```diff
@@ -203,7 +203,6 @@ Monitor::Monitor(CephContext* cct_, string nm, MonitorDBStore *s,
   timecheck_rounds_since_clean(0),
   timecheck_event(NULL),
 
-  paxos_service(PAXOS_NUM),
```

Comment:
better off change the prefix of the title to "mon/Monitor:". this change does not touch the paxos implementation.

Reply:
Done